bitnami/grafana-operator/values.yaml

```diff
@@ -552,6 +555,17 @@ grafana:
     timeoutSeconds: 5
     failureThreshold: 6
     successThreshold: 1
+  ## @param grafana.updateStrategy.type Set up update strategy for Grafana installation.
```

Comment:
There is an validation error on the  [metadata](https://github.com/bitnami-labs/readme-generator-for-helm#metadata) of the parameters in the values.yaml.

This should be:
```
  ## @param grafana.updateStrategy Set up update strategy for Grafana installation.
```